ui/src/app/shared/components/events-list/events-list.tsx

```diff
@@ -31,12 +40,30 @@ export const EventsList = (props: {events: models.Event[]}) => {
                                 <div className='columns small-4 xxlarge-5'>{event.message}</div>
                                 <div className='columns small-2 xxlarge-1'>{event.count}</div>
                                 <div className='columns small-2 xxlarge-2'>
-                                    <div className='events-list__event__ago'>{ago(new Date(event.firstTimestamp))}</div>
-                                    <div className='events-list__event__time'>{readableFormat(new Date(event.firstTimestamp))}</div>
+                                    {event.firstTimestamp ? (
+                                        <>
+                                            <div className='events-list__event__ago'>{ago(new Date(event.firstTimestamp))}</div>
+                                            <div className='events-list__event__time'>{readableFormat(new Date(event.firstTimestamp))}</div>
```

Comment:
I know this is preexisting code, but the idea of calling `new Date(x)` .. `new Date(x)` instead of `date_x=new Date(x)` .. `date_x` .. `date_x` bothers me.

Reply:
Hi @jsoref , that's fair.  Please have a look again.